packages/core/test/mécanismes/unité.yaml

```
@@ -15,6 +15,14 @@ conversion d'unité:
     - unité attendue: an
       valeur attendue: 1
 
+non applicable préserve l'unité:
+  formule:
```

```suggestion
  valeur:
```

Je serai pour déprécier `formule` à l'occasion.

---

J'ai gardé "formule" par cohérence avec les règles au-dessus.

À mon avis la suppression de "formule" mérite un ticket séparé :
- Comme mécanisme qui peut s'utiliser partout, "formule" n'a pas d'intérêt ce rôle étant déjà rempli par "valeur"
- Mais on avait décidé de le garder par soucis de lisibilité des règles https://github.com/betagouv/mon-entreprise/issues/836#issuecomment-575606622
- On pourrait le garder en premier niveau uniquement, si on avait la possibilité de définir des clés personnalisées de premier niveaux (exemple: "condition" pour une règle de type "notification" ou "formule" pour un calcul) https://github.com/betagouv/publicodes/discussions/196
- Ou alors décider de le supprimer complétement, mais alors au temps le faire partout et dans une PR dédiée

---

ah bizarrement quand on enlève "formule" ça casse le test 🤔